src/display.rs

```diff
@@ -91,23 +91,23 @@ where
         Command::DriverOutputControl(self.config.dimensions.rows, 0x00)
             .execute(&mut self.interface)?;
 
+        Command::GateDrivingVoltage(0x17).execute(&mut self.interface)?;
+        Command::SourceDrivingVoltage(0x41, 0xAC, 0x32).execute(&mut self.interface)?;
         self.config.dummy_line_period.execute(&mut self.interface)?;
         self.config.gate_line_width.execute(&mut self.interface)?;
+        self.config.data_entry_mode.execute(&mut self.interface)?;
 
-        // Command::GateDrivingVoltage(0b10000 | 0b0001);
-        // Command::SourceDrivingVoltage(0x2D, 0xB2, 0x22).execute(&mut self.interface)?;
         self.config.write_vcom.execute(&mut self.interface)?;
 
         // POR is HiZ. Need pull from config
-        // Command::BorderWaveform(u8).execute(&mut self.interface)?;
+        Command::BorderWaveform(0b00110001).execute(&mut self.interface)?;
```

Comment:
Instead of / in addition to calling this in `init`, it might be nice to expose this via a `set_border` function.